free-courses-en.md

```diff
@@ -187,7 +188,10 @@
 * [Docker Tutorial for Beginners](https://www.youtube.com/watch?v=fqMOX6JJhGo) - FreeCodeCamp
 * [Learn Docker, Container Runtimes, Builders and Registries](https://www.katacoda.com/courses/container-runtimes) - Katacoda
 
-
+### Flutter
+* [Flutter Tutorial](https://www.udacity.com/course/build-native-mobile-apps-with-flutter--ud905) - Udacity
+* [Flutter Tutorial for Beginners](https://www.udemy.com/course/learn-flutter-beginners-course/) - Udemy
+* [Flutter Crash Course](https://fluttercrashcourse.com/courses/basics/lessons/materialapp-scaffold-appbar-text) - Flutter CrashCourse
 ### Game Development
```

Comment:
There should also be 2 line breaks above this section.

Reply:
okk

Reply:
Perhaps it's best if you look at the rest of the document to get an idea in the formatting?

There is no need to add line breaks via `<br>`

```
* [Flutter Crash Course](https://fluttercrashcourse.com/courses/basics/lessons/materialapp-scaffold-appbar-text) - Flutter CrashCourse


### Game Development
```

This is also stated in the contribution guide.